frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/create-local-volume-set.tsx

```diff
@@ -53,6 +61,11 @@ export const CreateLocalVolumeSet: React.FC<CreateLocalVolumeSetProps> = ({
     'ceph-storage-plugin~Selecting all nodes will use the available disks that match the selected filters on all nodes selected on previous step.',
   );
 
+  const setOCSTaint = (nodes: string[]): void => {
```

Comment:
why not use the `React.useEffect` on nodes array?

Reply:
If you check, this is the handler function to set tainted nodes on the root state.
There are no nodes fetched in parent component.